examples/OCPS.ipynb

```diff
@@ -0,0 +1,341 @@
+{
```

Comment:
<p>I think you're the only one I've seen put this here, so it's definitely not expected. I think this functionality is going to be hidden shortly and then this info will be stale.</p><p>If I'm wrong or you want to preserve it maybe considering linking to where the info on the <code>setup.env</code>  is.</p><p>https://confluence.lsstcorp.org/display/LSSTCOM/Cycle+20+Upgrade</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Quite happy to leave this out since users shouldn't have to worry about it.